docs/domain/global.rst

```diff
@@ -0,0 +1,27 @@
+.. _labels_domain:
+
+Label Domain
+**************
+
+Labels are Key Value pairs.
+
+.. code-block:: python
+
+    label = {"key":"value"} # One Label
+    labels = [{"key":"value"},{"another_key":"value"}] # Many Labels
```

Comment:
Why are multiple labels not represented as one dictionary with multiple entries?

```{"key": "value", "another_key": "value"}```